src/components/EnvironmentBadge.js

```diff
@@ -1,30 +1,43 @@
 import React from 'react';
 import {Text, View} from 'react-native';
 import styles from '../styles/styles';
-import CONFIG from '../CONFIG';
 import CONST from '../CONST';
+import Environment from '../libs/Environment';
 
-const EnvironmentBadge = () => {
-    // If we are on production, don't show any badge
-    if (CONFIG.EXPENSIFY.ENVIRONMENT === CONST.ENVIRONMENT.PRODUCTION) {
-        return null;
-    }
+class EnvironmentBadge extends React.Component {
+    constructor() {
+        super();
```

Comment:
```suggestion
    constructor(props) {
        super(props);
```
Always need to pass the props through